checkov/common/output/record.py

```diff
@@ -193,8 +193,12 @@ def to_string(self, compact: bool = False, use_bc_ids: bool = False) -> str:
                         )
 
         status_message = colored("\t{} for resource: {}\n".format(status, self.resource), status_color)
-        secret_validation_status_string = f'\tStatus: {self.validation_status}\n' if \
-            self.check_result["result"] == CheckResult.FAILED and hasattr(self, 'validation_status') else ""
+
+        secret_validation_status_string = ""
+        if self.check_result["result"] == CheckResult.FAILED and \
+                hasattr(self, 'validation_status') and \
+                os.getenv("CKV_VALIDATE_SECRETS"):
```

Comment:
not so related to this PR, but should `CKV_VALIDATE_SECRETS` become a proper flag at some point? then we need to add it to `RunnerFilter` and then pass it via a parameter to this code part

Reply:
Nope, hopefully we will expose it soon. Not much overhead for regular secrets scanning @gruebel